packages/sku/telemetry/index.js

```diff
@@ -1,24 +1,32 @@
 const os = require('node:os');
 
-const { getPathFromCwd } = require('../lib/cwd');
+const { getPathFromCwd, cwd } = require('../lib/cwd');
 const isCI = require('../lib/isCI');
 const provider = require('./provider');
 const skuVersion = require('../package.json').version;
 
 const { languages } = require('../context');
 
 let projectName = 'unknown';
+let braidVersion = 'unknown';
 try {
   const packageJson = require(getPathFromCwd('package.json'));
 
   if (packageJson.name) {
     projectName = packageJson.name;
   }
+
+  const braidPackageJson = require(require.resolve(
+    'braid-design-system/package.json',
+  ), [cwd()]);
```

Comment:
```suggestion
  const braidPackageJson = requireFromCwd('braid-design-system/package.json');
```

👀 

https://github.com/seek-oss/sku/blob/721990812bb10d5c5879d19e15af29b652c5bbf7/packages/sku/lib/cwd.js#L25-L29

Reply:
Ooo, should've scouted this out, felt like I was re-implementing something that already existed.